src/ReinforcementLearningZoo/src/algorithms/nfsp/nfsp.jl

```diff
@@ -0,0 +1,190 @@
+"""
+Neural Fictitious Self-Play (NFSP) agent implemented in Julia.
+
+See the paper https://arxiv.org/abs/1603.01121 for more details.
+"""
```

Comment:
I believe this docstring should go to `NFSPAgents`. Please follow the Julia guideline https://docs.julialang.org/en/v1/manual/documentation/